index.d.ts

```diff
@@ -1,4592 +1,4 @@
-    avatar_url: string;
-    gravatar_id: string;
-    url: string;
-    html_url: string;
-    followers_url: string;
-    following_url: string;
-    gists_url: string;
-    starred_url: string;
-    subscriptions_url: string;
-    organizations_url: string;
-    repos_url: string;
-    events_url: string;
-    received_events_url: string;
-    type: string;
-    site_admin: boolean;
-  };
-  type WebhookPayloadOrganizationMembership = {
-    url: string;
-    state: string;
-    role: string;
-    organization_url: string;
-    user: WebhookPayloadOrganizationMembershipUser;
-  };
-  type WebhookPayloadOrganization = {
-    action: string;
-    membership: WebhookPayloadOrganizationMembership;
-    organization: WebhookPayloadOrganizationOrganization;
-    sender: WebhookPayloadOrganizationSender;
-  };
-  type WebhookPayloadMilestoneSender = {
-    login: string;
-    id: number;
-    node_id: string;
-    avatar_url: string;
-    gravatar_id: string;
-    url: string;
-    html_url: string;
-    followers_url: string;
-    following_url: string;
-    gists_url: string;
-    starred_url: string;
-    subscriptions_url: string;
-    organizations_url: string;
-    repos_url: string;
-    events_url: string;
-    received_events_url: string;
-    type: string;
-    site_admin: boolean;
-  };
-  type WebhookPayloadMilestoneMilestoneCreator = {
-    login: string;
-    id: number;
-    node_id: string;
-    avatar_url: string;
-    gravatar_id: string;
-    url: string;
-    html_url: string;
-    followers_url: string;
-    following_url: string;
-    gists_url: string;
-    starred_url: string;
-    subscriptions_url: string;
-    organizations_url: string;
-    repos_url: string;
-    events_url: string;
-    received_events_url: string;
-    type: string;
-    site_admin: boolean;
-  };
-  type WebhookPayloadMilestoneMilestone = {
-    url: string;
-    html_url: string;
-    labels_url: string;
-    id: number;
-    node_id: string;
-    number: number;
-    title: string;
-    description: string;
-    creator: WebhookPayloadMilestoneMilestoneCreator;
-    open_issues: number;
-    closed_issues: number;
-    state: string;
-    created_at: string;
-    updated_at: string;
-    due_on: string;
-    closed_at: null;
-  };
-  type WebhookPayloadMilestone = {
-    action: string;
-    milestone: WebhookPayloadMilestoneMilestone;
-    repository: PayloadRepository;
-    sender: WebhookPayloadMilestoneSender;
-  };
-  type WebhookPayloadMetaSender = {
-    login: string;
-    id: number;
-    node_id: string;
-    avatar_url: string;
-    gravatar_id: string;
-    url: string;
-    html_url: string;
-    followers_url: string;
-    following_url: string;
-    gists_url: string;
-    starred_url: string;
-    subscriptions_url: string;
-    organizations_url: string;
-    repos_url: string;
-    events_url: string;
-    received_events_url: string;
-    type: string;
-    site_admin: boolean;
-  };
-  type WebhookPayloadMetaHookConfig = 
```

Comment:
Is it ok to rely on a `lib/generated` folder from root level file? (even though the file is also generated?)

Reply:
Yes, sure! I depend on generated code in many of my libraries. I only put them in a `generated/` folder to clearly signal that these files are generated and should not be edited manually